distributed/bokeh/components.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, n_rectangles=1000, **kwargs):
         self.root = Plot(
             title=Title(text="Task Stream"), id='bk-task-stream-plot',
             x_range=x_range, y_range=y_range, toolbar_location="above",
-            min_border_right=35, **kwargs
+            min_border_right=35, plot_height=300, **kwargs
```

Comment:
I've removed it entirely.  This was only here as a hack to avoid the reset issue